src/main/java/cofh/thermaldynamics/duct/fluid/FluidGrid.java

```diff
@@ -245,7 +245,7 @@ public FluidStack getNodeShare(DuctUnitFluid theCond) {
 
 	public int getNodeAmount(DuctUnitFluid theCond) {
 
-		if (theCond.canStoreFluid()) {
+		if (!theCond.canStoreFluid()) {
```

Comment:
Already made this change in my dev branch locally like last week, just hadn't pushed. Not sure if it'll merge conflict or not.